src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -114,13 +114,23 @@ typedef struct {
     };
 } struct_80854B18; // size = 0x08
 
+/**
+ * Contains the rotation and angular velocity of the Bunny Hood's ears, which share the same x and y motion, but apply
+ * the z displacement in opposite directions. It is useful to keep in mind that
+ * - as usual, rot.x,rot.y,rot.z map to the Z,X,Y angles for the rotations applied, and
+ * - in its usual orientation, Player's head limb has x forwards, y downwards and z leftwards.
+ *
+ * Hence, in order, the rotations applied are:
+ * - rot.x (around the leftward axis, i.e. forward/backward)
+ * - rot.y (around the forward axis, i.e. left/right)
+ * - rot.z (around the vertical axis)
+ *
+ * @see Player_DrawGameplay(), Player_UpdateBunnyEars()
+ */
```

Comment:
Several things:
- "as usual, rot.x,rot.y,rot.z map to the Z,X,Y angles for the rotations applied": looks wrong, unless that's meant as a joke idk
- "in order, the rotations applied": the order used is input-space first, which is the opposite (would be fine if it was explicit, but I had to piece it together from figuring out the order all over again, see discord https://discord.com/channels/688807550715560050/1062655734338301952/1062665340821196831 )
- "* - in its usual orientation, Player's head limb has x forwards, y downwards and z leftwards.": this is true afaik but lacking an explanation of why it is relevant here (it is, I think)
- I don't fully understand this comment so I think it is of limited use anyway. I think it is trying to not go in-depth but it ends up not explaining anything useful

so I suggest removing this comment entirely.

FeelsBadMan to discard Elliptic's work again, but Elliptic if you read this someday feel free to PR docs again and we can actually discuss it. Another thing is I'm not sure you noticed `0x0B000000` is used with `G_MTX_MUL`. https://discord.com/channels/688807550715560050/1062655734338301952/1062677096733884447

Reply:
im fine with removing it, but I guess i should wait to see if anyone has anything to say in the discord thread?

Reply:
i removed the comment

I also moved the struct definition down right near the variable so its not floating 10k lines away at the top of the file (as other things get doc'd they will move to relevant places too)